cabal.project

```diff
@@ -219,8 +219,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger
-  tag: 52da70e5a0472cd4433876289f1aebaa0c6e5c85
-  --sha256: 0aiislbwx5yqdidwd66zqqpskvay84iwkgsgi5l96rbfcsf0n8lq
+  tag: 9d5fe3e539605e3bf9d98d73ea892aa467d55058
```

Comment:
```
$ git lg 52da70e5a0472cd4433876289f1aebaa0c6e5c85^..9d5fe3e539605e3bf9d98d73ea892aa467d55058
9d5fe3e53 - Merge pull request #2861 from input-output-hk/jc/do-not-emit-empty-SuccessfulPlutusScriptsEvent (31 hours ago) <Jared Corduan>
ff4f9f496 - Merge pull request #2860 from input-output-hk/jc/remove-babbage-minCollateral (2 days ago) <Jared Corduan>
3cc007d8b - Merge pull request #2863 from input-output-hk/jc/do-not-serialize-txbody-empty-fields (2 days ago) <Jared Corduan>
aaca526f8 - Merge pull request #2864 from input-output-hk/jc/fix-vrf-range-extension-in-babbage-ledger-spec (2 days ago) <Jared Corduan>
0b0d066bf - Merge pull request #2858 from input-output-hk/jc/unit-test-using-simple-script-as-reference-script (6 days ago) <Jared Corduan>
0836751c4 - Merge pull request #2857 from input-output-hk/td/test-validate-only-required-scripts (6 days ago) <teodanciu>
52da70e5a - Merge pull request #2852 from input-output-hk/jc/reference-scripts-not-always-executed (8 days ago) <Jared Corduan>
```

Your PR description mentions input-output-hk/cardano-ledger#2861 and input-output-hk/cardano-ledger#2863. It appears that 2864, 2858, and 2857 are just docs/tests.

What about 2860?

Reply:
yes, exactly! let me know, @nfrisby , if there is anything you'd like me to add to PR notes in future updates like this one. Maybe next time, when I omit mentioning commits related to documentation and testing, I can say as much. Thank you, as always, for being so thorough!!